lib/fpm/package/python.rb

```diff
@@ -310,6 +314,13 @@ def install_to_staging(setup_py)
         flags += [ "build_scripts", "--executable", attributes[:python_scripts_executable] ]
       end
 
+      if !attributes[:python_setup_py_arguments].nil? and !attributes[:python_setup_py_arguments].empty?
```

Comment:
this won't ever be nil because the default is `[]` (empty array), but there's probably no harm in this nil check.